pkg/frontend/security_test.go

```diff
@@ -119,27 +137,62 @@ func TestSecurity(t *testing.T) {
 			cert:           validclientcerts[0],
 			wantStatusCode: http.StatusNotFound,
 		},
+		{
+			name:           "empty url, valid admin certificate",
+			url:            "https://server/",
+			key:            validadminclientkey,
+			cert:           validadminclientcerts[0],
+			wantStatusCode: http.StatusForbidden,
+		},
 		{
 			name:           "unknown url, valid certificate",
 			url:            "https://server/unknown",
 			key:            validclientkey,
 			cert:           validclientcerts[0],
 			wantStatusCode: http.StatusNotFound,
 		},
+		{
+			name:           "unknown url, valid admin certificate",
+			url:            "https://server/unknown",
+			key:            validadminclientkey,
+			cert:           validadminclientcerts[0],
+			wantStatusCode: http.StatusForbidden,
+		},
 		{
 			name:           "operations url, valid certificate",
 			url:            "https://server/providers/Microsoft.RedHatOpenShift/operations?api-version=2019-12-31-preview",
 			key:            validclientkey,
 			cert:           validclientcerts[0],
 			wantStatusCode: http.StatusOK,
 		},
+		{
```

Comment:
Are we missing a test of "operations url, valid certificate" -> http.StatusForbidden?
i.e. validate that an admin cert holder cannot use the public API on a cluster.